graylog2-web-interface/src/components/configurations/UserConfig.tsx

```diff
@@ -0,0 +1,152 @@
+})(({ theme }: { theme: DefaultTheme }) => css`
+  &&.deflist {
+    dd {
+      padding-left: ${theme.spacings.md};
+      margin-left: 200px;
+    }
+  }
+`);
+
+const LabelSpan = styled.span(({ theme }: { theme: DefaultTheme }) => css`
+  margin-left: ${theme.spacings.sm};
+  font-weight: bold;
+`);
+
+const UserConfig = ({ config, updateConfig }: Props) => {
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const _saveConfig = (values) => {
+    updateConfig(values).then(() => {
+      setShowModal(false);
+    });
+  };
+
+  const _resetConfig = () => {
+    setShowModal(false);
+  };
+
+  const _timeoutIntervalValidator = (milliseconds: number) => {
+    return milliseconds >= 1000;
+  };
+
+  return (
+    <div>
+      <h2>User Configuration</h2>
+      <p>These settings can be used to set a global session timeout value.</p>
+
+      {!config ? <Spinner /> : (
+        <>
+          <StyledDefList>
+            <dt>Global session timeout:</dt>
+            <dd>{config.enable_global_session_timeout ? 'Enabled' : 'Disabled'}</dd>
+            <dt>Timeout interval:</dt>
+            <dd>{config.enable_global_session_timeout ? config.global_session_timeout_interval : '-'}</dd>
+          </StyledDefList>
+
+          <IfPermitted permissions="clusterconfigentry:edit">
+            <p>
+              <Button type="button"
+                      bsSize="xs"
+                      bsStyle="info"
+                      onClick={() => {
+                        setShowModal(true);
+                      }}>
+                Edit configuration
+              </Button>
+            </p>
+          </IfPermitted>
+
+          <Modal show={showModal} onHide={_resetConfig} aria-modal="true" aria-labelledby="dialog_label">
+            <Formik onSubmit={_saveConfig} initialValues={config}>
+
+              {({ isSubmitting, values, setFieldValue }) => {
+                return (
+                  <Form>
+                    <Modal.Header closeButton>
+                      <Modal.Title id="dialog_label">Update User Configuration</Modal.Title>
+                    </Modal.Header>
+
+                    <Modal.Body>
+                      <div>
+                        <Row>
+                          <Col sm={12}>
+                            <FormikInput type="checkbox"
+                                         name="enable_global_session_timeout"
+                                         id="enable_global_session_timeout"
+                                         label={(
+                                           <LabelSpan>Enable global session timeout</LabelSpan>
+                                         )} />
+                            <InputDescription help="If enabled, it will be set for all the users." />
+                          </Col>
+                          <Col sm={12}>
+                            <fieldset>
+                              <ISODurationInput id="global_session_timeout_interval"
```

Comment:
The validation is triggered when the field is disabled as you can see below:
![image](https://user-images.githubusercontent.com/1381455/213122206-7a0032cb-8ca0-48a0-bd5a-75fdf536e8a6.png)

The input has a default value `0` and when I remove it to enter a value a `P` appear. I think we should fix this with a proper default value.